### ISSUE TITLE
SBML files need observables to be defined as non-constant

### ISSUE THREAD

**Issue by LoosC** [ICB-DCM/AMICI#351]
I tried to import the model_steadystate_scaled.sbml file in Copasi. Two issues:
- need to manually rename it to .xml (shouldn't we use the standard file ending for SBML files?)
- for the observables_* it needs to be set: constant="true" (otherwise an error occurs in the import)

**Comment by thomassligon** [comment]
Hi Carolin,

I don’t think it’s necessary to rename the file. I started Copasi and selected File/Import SBML… and then navigated to the folder. In the window, after the “File name:” box there is a dropdown list that contains (by default) “XML Files (*.xml)” so I needed to change that to the other entry “All Files (*)” and then I can see model_steadystate_scaled.sbml and import it. However, COPASI has error messages, such as

>ERROR 2018-07-19 22:03:06<
SBML (34): Error: AssignmentRule is not allowed for local constant 'Parameter' identified by id 'observable_x1withsigma'.

I also get these errors when I open the file on
sbml.org/Facilities/Validator
so this is not correct SBML.

Tom
(Dr. Thomas S. Ligon)
Tom@thomassligon.info<mailto:Tom@thomassligon.info>
Frohnloher Str. 6a
81475 Muenchen
Germany
Tel. +49(89)74575075
